resources/js/processes/modeler/components/ModelerApp.vue

```diff
@@ -100,18 +100,18 @@
       });
       return notifications;
     },
-    actionSaveBpmn ({xml, svg}) {
+    emitRegisteredEvents ({xml, svg}) {
       this.dataXmlSvg.xml = xml;
       this.dataXmlSvg.svg = svg;
 
       this.externalEmit.forEach(item => {
         window.ProcessMaker.EventBus.$emit(item);
       })
-      if (this.saveEmitDefault) {
+      if (!this.externalEmit) {
```

Comment:
How is `externalEmit` modified? It seems it's set to an empty array and never modified. An empty array is truthy, so `!this.externalEmit` will always evaluate to `false`, so it seems this `if` block can be removed. Is that correct?

Reply:
Yes, this array is established by an external package, which records events.